cli/internal/packager/deploy.go

```diff
@@ -82,6 +82,16 @@ func Deploy() {
 	}
 
 	if config.IsZarfInitConfig() {
+		// Create a gitea org and a read-only user
+		err := git.CreateZarfOrg()
```

Comment:
These two operations should only happen if Gitea was listed as a component. I need to think about how the best way to figure that out would be.